hyph-utf8/tex/generic/hyph-utf8/patterns/tex/hyph-as.tex

```diff
@@ -1,28 +1,47 @@
-% These patterns originate from
-%    https://github.com/santhoshtr/hyphenation/)
-% and have been adapted for hyph-utf8 (for use in TeX).
+% title: Hyphenation for Assamese
+% copyright: Copyright (C) 2016 Santhosh Thottingal (santhosh dot thottingal at gmail dot com)
+% notice: This file is part of the hyph-utf8 project.
+%     See http://www.hyphenation.org for more information.
+%     Original source https://github.com/santhoshtr/hyphenation/
```

Comment:
The files with indic hyphenation patterns are "auto-generated" (or should I say "auto-updated") from upstream. We should fix the updating script as well.

Reply:
Covered by #9.